### DISCUSSION TITLE
pyMOR Community Meeting 2023-11-28

### DISCUSSION THREAD

**Discussion by sdrave** [pymor/pymor#2223] [Announcements]
The next pyMOR Community Meeting will take place Tuesday, November 28, at 15:00 CET and 9:00 EST, online at:

Link: http://bbb.mpi-magdeburg.mpg.de/meeting/pymor
Password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [x] 2023.2 release (@sdrave)
- [x] Future design direction (@sdrave)

Moderator: @sdrave

**Comment by lbalicki** [comment]
Meeting Notes:

- 2023.2 release
  - went through all PRs for milestone
  - remove string concatenation requirement in new linter options PR
  - went through Issues for milestone
- will from now on split assertions connected with "and" into multiple assertions
-  Future design direction / QR PR
  - proposed more fundamental discussion after holidays
  - put together a document from discussion
  - discuss how to proceed and when to do it at dev meeting next year
  - think about high-level questions like target audience of pyMOR, etc. to guide design

Next pyMOR Community Meeting will take place Tuesday, January 9, at 15:00 CET and 9:00 EST

Moderator: @pmli
